src/app.rs

```diff
@@ -272,9 +272,7 @@ lazy_static! {
         // The contents of the default public directory are included in the binary. The first time
         // the default directory is used, if ever, we extract them to a directory on the host file
         // system and return the path to that directory.
-        let path = dirs::data_local_dir()
-            .unwrap_or_else(|| env::current_dir().unwrap_or_else(|_| PathBuf::from("./")))
-            .join("tide-disco/public/media");
+        let path = PathBuf::from("$HOME/tide-disco/public/media");
```

Comment:
So `$HOME` get's expanded here? Neat!

I would make it a dotfile (`~/.tide-disco/`) so we aren't cluttering up people's home directories